Allow custom host/port for mqtt

efaden opened the pull request:
Allows for a custom host and port to be used.  If empty then uses the MQTT service variable.

efaden committed:
Allow custom host/port for mqtt

gordlea commented:
Thank you for the pr! I will review when I have a chance

gordlea commented:
I changed to using an override_mqtt_url config param so that the user can specify the protocol, if they are using mqtts or ws or wss etc. 

Also added the ability to specify custom username/password.

See gordlea/home-assistant-addons#54

Thanks again!

efaden commented:
Thanks!

On Tue, Mar 26, 2024, 11:09 Gord Lea ***@***.***> wrote:

> I changed to using an override_mqtt_url config param so that the user can
> specify the protocol, if they are using mqtts or ws or wss etc.
>
> Also added the ability to specify custom username/password.
>
> Thanks again!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/gordlea/home-assistant-addons/pull/53#issuecomment-2020693539>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADAEABCCMSWOJ6NDPFS2GSLY2GFRFAVCNFSM6AAAAABFHIIC7SVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDAMRQGY4TGNJTHE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
